mitogen/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: add EPEL repo on CentOS
+  yum:
+    name: epel-release
+    state: present
+  when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '7'
```

Comment:
The README says Amazon Linux 2 is also supported - is the epel repo already present in that distro?

Reply:
amazon linux 2 doesn't need epel for this role. it allows epel to be installed via "amazon-linux-extras install epel"

Reply:
@tequalsme I verified that Amazon Linux 2 includes the python2-pip and python3-pip packages in their core packages while CentOS 7 relies on the EPEL repo for those. I knew there was a reason I only installed EPEL on CentOS 7...I just couldn't remember why.